testnets/outbedevnet/chain.json

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "outbedevnet",
+  "chain_type": "cosmos",
+  "status": "live",
+  "website": "https://www.outbe.com",
+  "network_type": "devnet",
+  "pretty_name": "outbe-node",
+  "chain_id": "outbe-devnet-1",
+  "bech32_prefix": "outbe",
+  "daemon_name": "outbe-noded",
+  "node_home": "$HOME/.outbe-node",
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "unit",
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0,
+        "average_gas_price": 0.025,
+        "high_gas_price": 0.04
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "unit"
+      }
+    ],
+    "lock_duration": {
+      "time": "1814400s"
+    }
+  },
+  "codebase": {
+    "git_repo": "https://github.com/outbe/outbe-node",
+    "recommended_version": "v1.0.0",
```

Comment:
Error: Some version properties in codebase do not match the current version for outbedevnet.

Found mismatches:
Path: recommended_version, Reason: Missing in currentVersion, In "codebase": v1.0.0, In currentVersion: undefined
Path: compatible_versions, Reason: Missing in currentVersion, In "codebase": v0.9.0, In currentVersion: undefined

Please add recommended_version and compatible_versions to the object in the versions.json file.